### PR TITLE
Add Boba L2 (Avalanche) Network

### PR DESCRIPTION
**Pull request by dirtycajunrice**
Adds Boba's Avalanche L2 to the network list.
Official Docs link: https://docs.boba.network/for-developers/network-avalanche#boba-avalanche-l2-43288-for-the-avalanche-l1-43114

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -1566,5 +1566,19 @@
...
     "start": 401478,
     "logo": "ipfs://QmUdwAZJfyKGZnfPGDsCnNvGu123mdd57kTGj1Y3EWVuWK"
+  },
+  "43288": {
+    "key": "43288",
+    "name": "Boba Avax L2",
+    "shortName": "Boba-Avax",
+    "chainId": 43288,
+    "network": "mainnet",
+    "multicall": "0x352E11Da7C12EA2440b079A335E67ff9219f6FfB",
```
Comment (ChaituVR) [inline src/networks.json:1576]: Can verify and publish code here https://blockexplorer.avax.boba.network/address/0x352E11Da7C12EA2440b079A335E67ff9219f6FfB/contracts#address-tabs

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Every thing else looks good

File: src/networks.json (hunk 1 above)
Comment (dirtycajunrice) [reply src/networks.json:1576]: @ChaituVR  Whoops! sorry about that i could have swore i had done that. It is done now :)

Comment (ChaituVR) [reply src/networks.json:1619]: Thanks @dirtycajunrice

File: src/networks.json [hunk 2]
```diff
@@ -1609,5 +1609,19 @@
...
+    "rpc": [
+      "https://replica.avax.boba.network"
```
Comment (ChaituVR) [inline src/networks.json:1621]:
```suggestion
      "https://replica.avax.boba.network",
      "https://avax.boba.network"
```

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**